docs/url-file-uploads.md

```diff
@@ -0,0 +1,82 @@
+# URL File Uploads
+
+## Create a URL file upload
+
+To create a URL file upload, use the `CreateUrlFileUploadPayload` mutation.
+
+Example mutation:
+```graphql
+mutation {
+  CreateUrlFileUploadPayload(
+    file: Upload,
+    input: {
+      crawlType: "list-crawl-type",
```

Comment:
Please change to "list" as this is the only type we currently support in automator